src/Context/EmailContext.php

```diff
@@ -50,9 +50,6 @@ public function before(BeforeScenarioScope $event)
         // Also set through the 'supportbehat' extension
         // to ensure its available both in CLI execution and the tested browser session
         $this->mailer = new TestMailer();
-        Injector::inst()->registerService($this->mailer, Mailer::class);
```

Comment:
I'm not sure about removing this line specifically, I'd expect that it would 'survive', unlike the 2 lines below?

Reply:
I have no problems leaving that one in. I'll update the PR tomorrow